src/main/java/swettdg/com/CodeFellowship/models/ApplicationUser.java

```diff
@@ -17,12 +17,20 @@
 
     String username;
     String password;
+    String firstName;
+    String lastName;
+    String dob;
+    String bio;
 
     public ApplicationUser(){};
 
-    public ApplicationUser(String username, String password){
+    public ApplicationUser(String username, String password, String firstName, String lastName, String dob, String bio){

```

Comment:
add some getters to your ApplicationUser, or make the instance variables public.... else you're going to get errors when you try to access them